chain/vm/vm.go

```diff
@@ -73,7 +73,7 @@ func ResolveToKeyAddr(state types.StateTree, cst cbor.IpldStore, addr address.Ad
 
 	act, err := state.GetActor(addr)
 	if err != nil {
-		return address.Undef, aerrors.Newf(byte(exitcode.SysErrActorNotFound), "failed to find actor: %s", addr)
+		return address.Undef, aerrors.Newf(byte(exitcode.SysErrSenderInvalid), "failed to find actor: %s", addr)
```

Comment:
`SysErrSenderInvalid` is a weird error to return here, since this method is called in a few different places, not always with a message sender's address. 

I think something like `SysErrInternal` makes more sense?

Reply:
fixed